src/views/Home.jsx

```diff
@@ -30,26 +36,41 @@ export function Home({ createNewList, listToken, setListToken }) {
 	};
 
 	return (
-		<div>
+		<div className="container">
 			<div className="Home">
 				{listToken && <Navigate to={'/List'} replace={true} />}
-				<p>Welcome to your smart shopping list!</p>
-				<button onClick={handleClick}>Create New List</button>
+				<p className="welcome">Welcome to your smart shopping list!</p>
+				<Button variant="contained" onClick={handleClick}>
+					Create New List
+				</Button>
 			</div>
-			<div>
-				<h1>Join Existing List</h1>
-				<form onSubmit={handleSubmit}>
-					<label htmlFor="joinList">Enter List Name:</label>
-					<input
+			<div className="Home">
+				<p className="welcome">Join Existing List</p>
+				<form onSubmit={handleSubmit} id="joinList">
+					<TextField
+						id="outlined-basic joinList"
+						label="Enter List Name"
+						variant="outlined"
+						size="small"
+						sx={{ width: 360 }}
 						type="text"
 						name="joinList"
-						id="joinList"
 						value={token}
 						onChange={formHandler}
 					/>
-					<button>Join List</button>
+					<div className="actions">
+						<Button variant="contained" onClick={handleSubmit}>
+							Join
+						</Button>
+						<Button variant="outlined" type="reset" onClick={resetForm}>
+							Cancel
+						</Button>
+					</div>
 				</form>
 			</div>
```

Comment:
I think this could all get wrapped in the styling you have for #joinlist. This would help distinguish it from the welcome message. I've attached a sketch to show one option.
![HoneyDo-Edits](https://user-images.githubusercontent.com/49693755/203211462-3491f369-1d7f-45c6-af4c-8c87053461ba.jpg)

Reply:
Thanks @wheninseattle  for this great feedback. I was able to learn a bit more about MUI such as Typography, how to adjust padding and margin within the component, etc. This was fun! Also, I love your sketch, it made me realize just how bad my penmanship is. lol I've made all of the requested changes except setting a default width for the form. Just need direction on how to do this with MUI. Thanks again for the valuable feedback, I learned lots!

Reply:
Glad it was helpful! I'll have to tell my mom that someone likes my penmanship; she'll be so pleased!